repos/system_upgrade/el7toel8/libraries/config/tests/test_getenvvars.py

```diff
@@ -45,19 +31,19 @@ def test_get_product_type_valid(monkeypatch):
                   'LEAPP_DEVEL_TARGET_PRODUCT_TYPE': dst}
         exp_src = 'ga' if not src else src.lower()
         exp_dst = 'ga' if not dst else dst.lower()
-        monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(envars))
+        monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(envars=envars))
```

Comment:
the same like in https://github.com/oamg/leapp-repository/pull/489/files#r414407185

Reply:
What's the problem here? There's an argument, so we are creating a new instance here, not just using the default one.

Reply:
What's the problem here? There is an argument, so a new instance gets created instead of using the default one.

Reply:
Not sure already, since my link doesn't work:) Probably it is linked with the redudant envars var, which is used just to pass to CurrentActorMocked. Anyway this is really not important it leaving up to you to decide. From my pov foo(key=key) looks ugly